### ISSUE TITLE
Incorrect crtm-fix file on Orion

### ISSUE THREAD

**Issue by DavidHuber-NOAA** [JCSDA/spack-stack#1158]
**Describe the bug**
The CRTM fix file `/work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env-rocky9/install/intel/2021.9.0/crtm-fix-2.4.0.1_emc-qls55kd/fix/amsua_metop-c.SpcCoeff.bin` is incorrect in the new unified-env-rocky9 installation on Orion.  This is the same issue described in https://github.com/JCSDA/spack-stack/issues/1046.

**To Reproduce**
```bash
>diff -q /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env{,-rocky9}/install/intel/*/crtm-fix-2.4.0.1_emc-*/fix/amsua_metop-c.SpcCoeff.bin
Files /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env/install/intel/2022.0.2/crtm-fix-2.4.0.1_emc-ezbyzji/fix/amsua_metop-c.SpcCoeff.bin and /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env-rocky9/install/intel/2021.9.0/crtm-fix-2.4.0.1_emc-qls55kd/fix/amsua_metop-c.SpcCoeff.bin differ
```

**System:**
Orion

**Additional context**
This prevents the GSI from being ported to Rocky9 on Orion (https://github.com/NOAA-EMC/GSI/issues/754).

**Comment by RatkoVasic-NOAA** [comment]
Fixed:
`cmp /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env{,-rocky9}/install/intel/*/crtm-fix-2.4.0.1_emc-*/fix/amsua_metop-c.SpcCoeff.bin`

**Comment by DavidHuber-NOAA** [comment]
Great, thank you @RatkoVasic-NOAA!  Confirmed the files are identical.
